### DISCUSSION TITLE
Inconsistent Collision Shape Naming

### DISCUSSION THREAD

**Discussion by ghost** [bulletphysics/bullet3#3520] [General]
I'm using `btCollisionShape::getName()` in my GUI to determine the type of collision shape.

[`btSphereShape`](https://github.com/bulletphysics/bullet3/blob/master/src/BulletCollision/CollisionShapes/btSphereShape.h#L56) returns **"SPHERE"** (all caps), while [`btBoxShape`](https://github.com/bulletphysics/bullet3/blob/master/src/BulletCollision/CollisionShapes/btBoxShape.h#L274) returns **"Box"** (title case). Not a huge issue, but any reason for the discrepancy?

**Comment by erwincoumans** [comment]
The caps-lock key could have been stuck :-) If it bothers you, just send a pull request and we fix it, and cross the fingers no-one relies on the capitalization of the "SPHERE" (they should use getShapeType() instead).
